### DISCUSSION TITLE
Pre Version 10 Delta Green

### DISCUSSION THREAD

**Discussion by Mystara1** [deltagreen-foundryvtt/delta-green-foundry-vtt-system#68] [General]
Hello. My pathfinder game (with the modules I use) won't run in version 10. I've reverted back to version 9 for now. My question is I'd like to install Delta Green (which I believe worked on version 9) but is only available for version 10. Is there a way or place to find your Delta Green system v 1.0.5 which I believe worked on version 9. Thanks

**Comment by Mystara1** [comment]
Sorry to be a bother, I figured it out. Just had to download version 1.0.5, extract it and rename the directory to what the system.json wanted. Then drop it in the system directory.

**Comment by ealsandor** [comment]
Is there anyway we can get item transfer to work between actors... its such a nice quality of life change
